test/test_autograd.py

```diff
@@ -1485,6 +1485,36 @@ def backward(ctx, grad_output):
         out.sum().backward()
         self.assertEqual(x.grad.data, y_data)
 
+    def test_cat(self):
```

Comment:
I'm impressed that this is the only function that doesn't have a method, that didn't already have a standalone test :)